mypy.ini

```diff
@@ -14,3 +14,9 @@ ignore_missing_imports = True
 
 [mypy-posthog]
 ignore_missing_imports = True
+
+[mypy-pydantic]
```

Comment:
not sure if there was another way to overcome mypy issues (it was angry at whichever version wasn't installed for my when I ran it)